src/main/java/uk/gov/hmcts/divorce/solicitor/event/page/SolHowDoYouWantToApplyForDivorce.java

```diff
@@ -38,22 +38,18 @@ public void addTo(final PageBuilder pageBuilder) {
                 .readonly(LabelContent::getTheApplicantsOrApplicant1s, NEVER_SHOW)
                 .readonly(LabelContent::getRespondentsOrApplicant2s, NEVER_SHOW)
             .done()
-            .label("solHowDoYouWantToApplyForDivorcePara-1",
-                "The applicant can apply for the divorce on their own (as a 'sole applicant') or with their husband "
-                    + "or wife (in a 'joint application').\n\n"
-                    + "### Applying as a sole applicant\n\n"
-                    + "If the applicant applies as a sole applicant, the applicant's husband or wife responds to the divorce "
-                    + "application after you have submitted it.  The applicant will be applying on their own.\n\n"
-                    + "### Applying jointly, with the applicant's husband or wife\n\n"
-                    + "If the applicant applies jointly, the applicant's husband or wife joins and reviews this online "
-                    + "application before it's submitted. They will be applying together.\n\n"
-                    + "*How the applicant divides their money and property is dealt with separately. It should not affect "
-                    + "the decision on whether to do a sole or a joint application.*")
-            .mandatory(CaseData::getApplicationType, null, null,
-                "How does the applicant want to apply for the divorce?",
-                "Applicant 2 must agree with a joint application in its entirety.")
-            .mandatoryWithLabel(CaseData::getDivorceOrDissolution,
-                "Is the application for a divorce or dissolution?");
+            .mandatory(CaseData::getDivorceOrDissolution, null, null, " ")
```

Comment:
could use mandatoryWithLabel(CaseData::getDivorceOrDissolution, " ")

Reply:
Thanks Shaun - I was experimenting with the different methods and forgot to go back to ...WithLabel